recipe/meta.yaml

```diff
@@ -30,6 +30,7 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - {{ compiler('cuda') }}  # [cuda_compiler != "None" and cuda_compiler_version != "None"]
+    - sysroot_{{ target_platform }} 2.17  # [linux]
```

Comment:
Pinned to `sysroot` to address issue: https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/5643

This will eventually be resolved upstream: https://github.com/conda-forge/linux-sysroot-feedstock/issues/58

Though this workaround should be ok in any event